jpa-entity-generator-maven-plugin/pom.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would like to detect breaking changes on the core library side which may affect the maven plugin. That's the reason why we have the maven plugin classes in the project (we can detect build failures by `gradle test`). If we can detect build failures on the maven plugin side in CI builds, I'm willing to change the structure of the project.